### PR TITLE
Jordan grewe

### PR DESCRIPTION
**Pull request by amiangshu**

### CODE REVIEW

File: Env_Setup_Instructions [hunk 1]
```diff
@@ -0,0 +1,20 @@
+sudo apt-get install python3-venv -y
```
Comment (amiangshu) [inline Env_Setup_Instructions:1]: Could you please add a comment on top?

File: Env_Setup_Instructions [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+pip3 install rpi_281x
+pip3 install adafruit-circuitpython-neopixel
+python3 -m pip install --force-reinstall adafruit-blinka
+
+install zip to correct venv directory
+(zip can be found: https://core-electronics.com.au/guides/raspberry-pi/fully-addressable-rgb-raspberry-pi/)
+
+sudo su to run as root
+
+run strandtest
```
Comment (amiangshu) [inline Env_Setup_Instructions:20]: Another comment

File: rpi_scroll_message.py [hunk 3]
```diff
@@ -0,0 +1,658 @@
...
+LED_COUNT      = 768      # Number of LED pixels.
+LED_PIN        = 18      # GPIO pin connected to the pixels (18 uses PWM!).
+#LED_PIN        = 10      # GPIO pin connected to the pixels (10 uses SPI /dev/spidev0.0).
+LED_FREQ_HZ    = 800000  # LED signal frequency in hertz (usually 800khz)
+LED_DMA        = 10      # DMA channel to use for generating signal (try 10)
+LED_BRIGHTNESS = 65     # Set to 0 for darkest and 255 for brightest
+LED_INVERT     = False   # True to invert the signal (when using NPN transistor level shift)
+LED_CHANNEL    = 0       # set to '1' for GPIOs 13, 19, 41, 45 or 53
+
+
```
Comment (amiangshu) [inline rpi_scroll_message.py:21]: This is a comment

**Review by amiangshu — COMMENTED** [review]
Overall looks good. Just some NIT, other than LGTM.

**Review by Grewestr — COMMENTED** [review]
comment example

File: rpi_scroll_message.py (hunk 3 above)
Comment (TKosinski24) [reply rpi_scroll_message.py:21]: Issue Fixed

**Review by Grewestr — APPROVED** [review]
LGTM

**Review by TKosinski24 — APPROVED** [review]
Everything has been addressed.

**Pull request closed without merging**